src/constants.js

```diff
@@ -13,6 +13,10 @@ import {
     displayOS
 } from './Components/RosTable/RenderColumn';
 
+// Pagination
+export const PER_PAGE = 10;
+export const PAGE = 1;
+
```

Comment:
@r14chandra are these initial counts going to be same across all screens?

Reply:
Yes